packages/extension-ui/src/components/NextStepButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import arrowLeft from '../assets/arrowLeft.svg';
+import Svg from './Svg';
+import Button from './Button';
+
+type Props = React.ComponentProps<typeof Button>;
+
+export default function NextStepButton ({ children, ...props }: Props): React.ReactElement<Props> {
+  return (
+    <Button {...props}>
+      {children}
+      <ArrowRight src={arrowLeft}/>
```

Comment:
I looks a bit strange with the right... left. Perhaps we could have 2 simple react components <ArrowLeft /> and <ArrowRight /> with the transform hidden inside. And then apply additional styling upon usage.